common/utils.js

```diff
@@ -28,15 +28,82 @@ export async function buildConstantByNpy(builder, url) {
   if (!dataTypeMap.has(npArray.dataType)) {
     throw new Error(`Data type ${npArray.dataType} is not supported.`);
   }
-  const dimensions = npArray.shape;
+  let dimensions = npArray.shape;
   const type = dataTypeMap.get(npArray.dataType).type;
   const TypedArrayConstructor = dataTypeMap.get(npArray.dataType).array;
-  const typedArray = new TypedArrayConstructor(sizeOfShape(dimensions));
+  let typedArray = new TypedArrayConstructor(sizeOfShape(dimensions));
   const dataView = new DataView(npArray.data.buffer);
   const littleEndian = npArray.byteOrder === '<';
   for (let i = 0; i < sizeOfShape(dimensions); ++i) {
     typedArray[i] = dataView[`get` + type[0].toUpperCase() + type.substr(1)](
         i * TypedArrayConstructor.BYTES_PER_ELEMENT, littleEndian);
   }
+  // TODO(Wanming): This is a workaround to transpose 'ihwo' to 'hwio',
+  // and will be removed once 'ihwo' filterLayout is supported.
+  if (isDepthwise) {
+    const a = tf.tensor(typedArray, dimensions, type);
+    const b = tf.transpose(a, [1, 2, 0, 3]);
+    const buffer = await b.buffer();
+    dimensions = b.shape;
+    typedArray = buffer.values;
+    tf.dispose();
```

Comment:
I understand you need to dispose both `a` and `b`. However, probably it's good to wrap these code section into [`tf.tidy`](https://js.tensorflow.org/api/latest/#tidy).